e2e/test/scenarios/native/native.cy.spec.js

```diff
@@ -58,12 +55,21 @@ describe("scenarios > question > native", () => {
         "have.text",
         "Third collection",
       );
+      cy.log("after selecting a dashboard, it should be the new suggestion");
+      cy.findByLabelText(/Where do you want to save this/).click();
+    });
 
-      cy.button("Cancel").click();
+    H.entityPickerModal().within(() => {
+      cy.findByText("Orders in a dashboard").click();
+      cy.button("Select this dashboard").click();
```

Comment:
This is a bit of a quirk... you don't actually need to save something to a dashboard to have it be the "last selected thing", you just need to select it via the entity picker.

However, this is a nice secondary assertion that selecting a dashboard from the picker will post to `api/activity/recents`. But I could be convinced that this isn't the best way to do this